examples/js/loaders/OBJLoader2.js

```diff
@@ -221,7 +220,7 @@ THREE.OBJLoader2 = (function () {
 		}
 
 		Parser.prototype.setDebug = function ( debug ) {
-			this.debug = ( debug == null ) ? this.debug : debug;
+			this.debug = Boolean( debug ) ? debug : this.debug;
```

Comment:
In my opinion this is overcomplicating it. I would just do `debug === true` or `debug === false`. I think the code logic is easier to read that way.